Main/Model/control.cpp

```diff
@@ -21,6 +20,38 @@ Control::Control(ModelItem *parent, Value* controlNode) : ModelItem(parent, "con
 
 
 
+void Control::externalUpdate(QStringRef pointer, PatchFragment& patch) {
+    // The ID is assumed to never change.
+    QString initialValueStr = "/initial_value";
+    QString overrideAutomationStr = "/override_automation";
+    QString minimumStr = "/minium";
+    QString maximumStr = "/maximum";
+    QString stepStr = "/step";
+    // TODO: control symbol, connection
+
+    if (pointer.startsWith(initialValueStr)) {
+        float val = patch.patch["value"].GetFloat();
+        initialValue = val;
+        ui_currentValue = val;
+        emit displayValueChanged(ui_currentValue);
+    }
+    else if (pointer.startsWith(overrideAutomationStr)) {
+        overrideAutomation = patch.patch["value"].GetBool();
+    }
+    else if (pointer.startsWith(minimumStr)) {
+        minimum = patch.patch["value"].GetBool();
```

Comment:
Should probably be `GetFloat()` here and in the following two uses of it?

Reply:
Yeppp